requirements-docs.txt

```
@@ -11,3 +11,4 @@ sphinx-sitemap
 sphinx-togglebutton
 sphinxcontrib-spelling
 sphinxext-opengraph
+sphinxcontrib.video
```

The correct package name is `sphinxcontrib-video`, whereas the extension name is `sphinxcontrib.video`. I know...

Then, of course, you can either do a `pip install sphinxcontrib-video` or any `make` command to install it.

```suggestion
sphinxcontrib-video
```